modules/react/src/components/render-player.tsx

```diff
@@ -4,11 +4,11 @@
 import React, {useEffect, useState} from 'react';
 import {parse} from '@loaders.gl/core';
 import {ZipLoader} from '@loaders.gl/zip';
-import styled from 'styled-components';
+import {styled} from 'styled-components';
 import {type Encoders, GIF, JPEG, PNG, WEBM} from './encoders';
 
 const parseImages = async (blob: Blob, encoder?: Encoders): Promise<Record<string, string>> => {
-  const images = await parse(blob, ZipLoader);
+  const images = (await parse(blob, ZipLoader)) as Record<string, ArrayBuffer>;
```

Comment:
```suggestion
  const images = await parse(blob, ZipLoader);
```

Pretty sure this is typed OOB, but will revert if not